support/json/json_x86.go

```diff
@@ -3,9 +3,42 @@
 package json
 
 import (
+	"encoding/json"
+
 	"github.com/bytedance/sonic"
+
+	"github.com/goravel/framework/contracts/support"
 )
 
+type Json struct {
```

Comment:
The struct is fine, but I think we may need to move it to the `foundation` folder, because we need to initialize the JSON interface there, then `support/json` can use the implementation. I'm not sure if there is any problem, please try it first.